build/scoring_test.go

```diff
@@ -77,8 +77,8 @@ func TestBM25(t *testing.T) {
 			query:    &query.Substring{Pattern: "example"},
 			content:  exampleJava,
 			language: "Java",
-			// keyword-score:1.63 (sum-tf: 6.00, length-ratio: 2.00)
-			wantScore: 1.63,
+			// keyword-score:1.69 (sum-tf: 7.00, length-ratio: 2.00)
+			wantScore: 1.69,
```

Comment:
Wondering why I didn't catch this 🤦

Reply:
Should we add more info to the debug message? We could print the term frequency map instead of sum-tf. This wouldn't have helped for this test case, because we are just searching for one term, but for a more complex query it might be easier to debug.